pod-node/module.js

```diff
@@ -0,0 +1,8 @@
+const addon = require('./build/Release/module');
+
+const USERNAME = "some_username";
```

Comment:
This won’t be hard coded in the plugin, right? I guess the user will specify this at least once?

Reply:
surely. this is for now, just for testing purposes.